onnxruntime/core/graph/graph.cc

```diff
@@ -1406,7 +1406,7 @@ Status Graph::VerifyNoDuplicateName() {
 
     if (!node_name.empty() && node_name_to_index.end() != node_name_to_index.find(node_name)) {
       // The node has name and its name was used by another node.
-      Status status(ONNXRUNTIME, FAIL,
+      Status status(ONNXRUNTIME, onnxruntime::common::StatusCode::FAIL,
```

Comment:
is there another `FAIL` that this conflicts with now?
FYI, there's also ORT_MAKE_STATUS() which allows usage of the unqualified name
https://github.com/microsoft/onnxruntime/blob/e3c85d4262ff9dc3675abf826d5f5cd7f76517aa/include/onnxruntime/core/common/common.h#L191

Reply:
This FAIL code is in conflict with the one from ONNX repo.